json_schema/module/biomaterial/human_specific.json

```
@@ -33,6 +38,57 @@
             },
             "user_friendly": "Ethnicity",
             "guidelines": "Enter one or more ethnicities, separated by a comma."
+        },
+        "ethnicity_question": {
+            "description": "Question and response options posed to donors for collecting metadata on donor self-reported ethnicity and / or tribe and / or dialect group and / or race etc.",
+            "type": "string",
+            "user_friendly": "Ethnicity question",
+            "example": "What is your ethnicity?; Are you Hispanic/Latino?; Which categories describe you? Select all that apply. Note You may select more than one group. 1. American Indian or Alaska Native (for example, Aztec, Blackfeet Tribe, Mayan, Navajo Nation, Native Village of Barrow (Utqiagvik) Inupiat Traditional Government, Nome Eskimo Community, etc.), 2 - Asian (for example, Asian Indian, Chinese, Filipino, Japanese, Korean, Vietnamese, etc.), 3 - Black, African American, or African (for example, African American, Ethiopian, Haitian, Jamaican, Nigerian, Somali, etc.), 4 - Hispanic, Latino, or Spanish (for example, Columbian, Cuban, Dominican, Mexican or Mexican American, Puerto Rican, Salvadoran, etc.), 5 - Middle Eastern or North African (for example, Algerian, Egyptian, Iranian, Lebanese, Moroccan, Syrian, etc.), 6 - Native Hawaiian or other Pacific Islander (for example, Chamorro, Fijian, Marshallese, Native Hawaiian, Tongan, etc.), 7 - White (for example, English, European, French, German, Irish, Italian, Polish, etc.), 8 - None of these fully describe me (optional free text answer), 9 - Prefer not to answer",
+            "bionetworks": ["genetic diversity"]
+        },
+        "ethnicity_parents": {
```

```suggestion
        "parental_ethnicities": {
```

Just a suggestion

---

Or, in line with my [comment](https://github.com/HumanCellAtlas/metadata-schema/pull/1611#discussion_r2388487160) below

```suggestion
        "ethnicity_of_parents": {
```